certbot-dns-rfc2136/certbot_dns_rfc2136/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""
+The `~certbot_dns_rfc2136.dns_rfc2136` plugin automates the process of
+completing a ``dns-01`` challenge (`~acme.challenges.DNS01`) by creating, and
+subsequently removing, TXT records using RFC 2136 Dynamic Updates.
+
+
+Named Arguments
+---------------
+
+===================================== =====================================
+``--dns-rfc2136-credentials``         RFC 2136 credentials_ INI file.
+                                      (Required)
+``--dns-rfc2136-propagation-seconds`` The number of seconds to wait for DNS
+                                      to propagate before asking the ACME
+                                      server to verify the DNS record.
+                                      (Default: 60)
+===================================== =====================================
+
+
+Credentials
```

Comment:
I found the `update-policy { grant keyname. name _acme-challenge.host.example.com. txt; };` example you gave helpful when setting up BIND on my system.

Perhaps it would be useful to mention and/or link to something about that in a BIND-specific subsection of this Credentials section, so that users know the minimum necessary privileges they need to grant to the key?